docs/standard/datetime/system-text-json-support.md

```diff
@@ -224,6 +226,8 @@ The following levels of granularity are defined for formatting:
 
         Used to format a <xref:System.DateTime> or <xref:System.DateTimeOffset> with fractional seconds and with a local offset.
 
+    This level of granularity is compliant with [RFC 3339](https://tools.ietf.org/html/rfc3339#section-5.6).
```

Comment:
```suggestion
This level of granularity is compliant with [Internet Date/Time Format from RFC 3339](https://tools.ietf.org/html/rfc3339#section-5.6).
```

Reply:
Similar to https://github.com/dotnet/docs/pull/21264/files#r514584458, this note only applies to the `'Date time'` format, so I think the indentation is correct here.